RecoParticleFlow/PFSimProducer/plugins/ConvBremSeedProducer.cc

```diff
@@ -398,29 +417,23 @@ void ConvBremSeedProducer::produce(Event& iEvent, const EventSetup& iSetup) {
 }
 
 void ConvBremSeedProducer::beginRun(const edm::Run& run, const EventSetup& iSetup) {
-  ESHandle<GeometricSearchTracker> track;
-  iSetup.get<TrackerRecoGeometryRecord>().get(track);
+  auto track = iSetup.getHandle(geomSearchTrackerToken_);
```

Comment:
```suggestion
  geomSearchTracker_ = iSetup.getData(geomSearchTrackerToken_);
```
(and the same for all ESProducts below...